File: app/src/main/java/org/tat/sdksample/Application.java
package org.tat.sdksample;

import org.th.tatsdk.TATSDKEnvironment;

public class Application extends android.app.Application {

    @Override
    public void onCreate() {
        super.onCreate();
        String apiKey = "[API_KEY]";
        TATSDKEnvironment.setEnvironment(apiKey, this);
    }
}

File: app/src/main/java/org/tat/sdksample/Application.kt
package org.tat.sdksample

import org.th.tatsdk.TATSDKEnvironment

class Application : android.app.Application() {

    override fun onCreate() {
        super.onCreate()
        val apiKey = "[API_KEY]"
        TATSDKEnvironment.setEnvironment(apiKey, this)
    }
}